Dungeon Crawler/Assets/Scripts/BattleManager.cs

```diff
@@ -56,6 +59,7 @@ public void StartBattle(IBattleEntity player, IBattleEntity enemy)
         Player = player;
         Enemy = enemy;
 
+        BattleStart.Invoke();
```

Comment:
Try BattleStart?.Invoke();
If there are no methods subscribed to this event it will be ignored.

Reply:
Might be what breaks the tests